site/k8s_setup.md

```diff
@@ -110,6 +110,10 @@ $ rm -f generated/instances_id_rsa && terraform output ssh_private_key > generat
 
 8. If you need shared storage between your Kubernetes worker nodes, enable and configure NFS:
 
+In the current GA version, OCI Container Engine for Kubernetes support network Block storage that can be shared across nodes with access permission RWOnce (meaning that only one can write the rest only read). The WebLogic on Kubernetes domain created by the WebLogic Server Kubernetes Operator at this time requires a shared file system to store the WebLogic domain configuration, which MUST be accessible from all the pods across the nodes. As a workaround, we need to install an NFS server in one node and share the file system across all nodes.
```

Comment:
+In the current GA version, the OCI Container Engine for Kubernetes supports network block storage that can be shared across nodes with access permission RWOnce (meaning that only one can write, others can read only). At this time, the WebLogic on Kubernetes domain created by the WebLogic Server Kubernetes Operator, requires a shared file system to store the WebLogic domain configuration, which MUST be accessible from all the pods across the nodes. As a workaround, you need to install an NFS server on one node and share the file system across all the nodes.

Reply:
Did a slight re-write (clean-up) on both paragraphs.